### PR TITLE
Modernise dots for AST/SCH. Added prettier icons for AST Arcana Logs.

### PR DESCRIPTION
**Pull request by suShirow**
Modernise dots for AST/SCH. Added jump links to timeline on the timestamps. Added "Back to top" button for Arcana Logs.

### CODE REVIEW

**Comment by suShirow** [comment]
I have time-traveled from the past to tell you that this PR is ready to go

File: src/parser/jobs/ast/modules/ArcanaSuggestions.js [hunk 1]
```diff
@@ -2,28 +2,35 @@ import {t} from '@lingui/macro'
...
-import Module from 'parser/core/Module'
-import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {ActionLink} from 'components/ui/DbLink'
+import styles from './ArcanaSuggestions.module.css'
 
-import {DRAWN_ARCANA_USE, HELD_ARCANA_USE, SPEAR_USED} from './ArcanaGroups'
-import DISPLAY_ORDER from './DISPLAY_ORDER'
+const LADY_OF_CROWNS_STATUS_ICON = 'https://xivapi.com/i/014000/014840.png'
+const LORD_OF_CROWNS_STATUS_ICON = 'https://xivapi.com/i/014000/014841.png'
+const UNKNOWN_CARD_STATUS_ICON = 'https://xivapi.com/i/010000/010205.png'
```
Comment (ackwell) [inline src/parser/jobs/ast/modules/ArcanaSuggestions.js]: Guessing these are the PVP icons or similar?

File: src/parser/jobs/ast/modules/ArcanaSuggestions.js [hunk 2]
```diff
@@ -158,13 +166,24 @@ export default class ArcanaSuggestions extends Module {
 
 						{this.cardLogs.map(artifact => {
 							return <Table.Row key={artifact.timestamp} className={styles.cardActionRow}>
-								<Table.Cell>{this.parser.formatTimestamp(artifact.timestamp)}</Table.Cell>
+
+								<Table.Cell>
+									<a onClick={() =>
+										this.timeline.show((artifact.timestamp-timelineLinkLowerMod) - this.parser.fight.start_time, (artifact.timestamp+timelineLinkUpperMod) - this.parser.fight.start_time)}>
```
Comment (ackwell) [inline src/parser/jobs/ast/modules/ArcanaSuggestions.js]: would be good to get this callback out of the jsx

File: src/parser/jobs/ast/modules/Combust.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+			requirements: [
+				new Requirement({
+					name: <Trans id="ast.dots.requirement.uptime.name"><ActionLink {...ACTIONS.COMBUST_II} /> uptime</Trans>,
+					percent: () => this.getUptimePercent(STATUSES.COMBUST_II.id),
+				}),
+			],
+		}))
+	}
+
+	addClippingSuggestions(clip: any) {
```
Comment (ackwell) [inline src/parser/jobs/ast/modules/Combust.tsx]: Given it's `any` due to core being JS, probably better off typing it as `TODO`

File: src/parser/jobs/sch/modules/DoTs.js [hunk 4]
```diff
@@ -1,78 +1,39 @@
 import React, {Fragment} from 'react'
```
Comment (ackwell) [inline src/parser/jobs/sch/modules/DoTs.js]: I didn't realise this was both sch and ast for a sec there and got super confused :notlikemeow:

File: src/parser/jobs/ast/modules/ArcanaSuggestions.js (hunk 1 above)
Comment (suShirow) [reply src/parser/jobs/ast/modules/ArcanaSuggestions.js]: Yeah they look more consistent

File: src/parser/jobs/ast/modules/ArcanaSuggestions.js (hunk 2 above)
Comment (suShirow) [reply src/parser/jobs/ast/modules/ArcanaSuggestions.js]: I concur

**Comment by suShirow** [comment]
Ready to go through the ack'machine again

**Pull request merged**